_example/simple/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# =============================================================================
+#  Multi-stage Dockerfile Example
+# =============================================================================
+#  This is a simple Dockerfile that will build an image of scratch-base image.
+#  Usage:
+#    docker build -t simple:local . && docker run --rm simple:local
+# =============================================================================
+
+# -----------------------------------------------------------------------------
+#  Build Stage
+# -----------------------------------------------------------------------------
+FROM golang:alpine AS build
+
+RUN apk add --no-cache \
+    # Build essentials for development on Alpine
+    alpine-sdk build-base \
+    # go-sqlite3 requirement
+    sqlite-dev
```

Comment:
I'm confused. This library statically links against its vendored copy of SQLite by default. I don't think this is required.

Reply:
Indeed. The minimum requirement is `gcc` (and `musl-dev` in Alpine).

My intention was to tell people to use build-essentials-for-dev for each distribution before issuing, as I have seen many issues with people forgetting to install the C compiler.

However, this seems to cause more confusion. I will slim it as much as possible.